src/baldor/__init__.py

```diff
@@ -9,9 +9,9 @@
 X_AXIS = np.array([1., 0., 0.], dtype=np.float64)
 Y_AXIS = np.array([0., 1., 0.], dtype=np.float64)
 Z_AXIS = np.array([0., 0., 1.], dtype=np.float64)
-_MAX_FLOAT = np.maximum_sctype(np.float)
+_MAX_FLOAT = np.maximum_sctype(np.float64)
 # epsilon for testing whether a number is close to zero
-_FLOAT_EPS = np.finfo(np.float).eps
+_FLOAT_EPS = np.finfo(np.float64).eps
 _EPS = np.finfo(float).eps * 4.0
```

Comment:
Looks like this line might need fixing as well?

Reply:
I though so as well, but that is actually using the build in float type.  Not sure why both are used, it seems redundant.